7.2.цикл_for_функция_range/7.2.10.py

```diff
@@ -0,0 +1,9 @@
+num = int(input())
+num1 = int(input())
+for i in range(num, num1 + 1):
+    if i % 17 == 0:
+        print(i)
+    elif i % 10 == 9:
+        print(i)
+    elif i % 15 == 0:
+        print(i)
```

Comment:
Тут можно и нужно использовать логическое ИЛИ:
```suggestion
    if i % 17 == 0 or i % 10 == 9 or i % 15 == 0:
        print(i)
```